lib/groupBy/groupBy.performance.js

```diff
@@ -0,0 +1,25 @@
+module.exports = function getObject() {
+  const length = 100000
+  const list1 = []
+  const list2 = []
+  const list3 = []
+
+  const func = (x) => x % 2 === 0 ? 'even' : 'odd'
+
+  for (var i = 0; i < length; i++) {
+    if (i < 100) list1.push(i)
+    if (i < 10000) list2.push(i)
+    list3.push(i)
+  }
```

Comment:
you should choose either `1e9, 1e8` and `1e7` and use `type: array_size` or change type and use `1e5, 1e4` and `1e3`